salt/map.jinja

```diff
@@ -17,6 +14,10 @@
 {% set osmap = salt['grains.filter_by'](osmap, grain='os') or{} %}
 {% do salt['defaults.merge'](defaults['salt'], osmap) %}
 
+{# merge the lookup #}
+{% set lookup = salt['pillar.get']('salt:lookup', default={}, merge=True) %}
+{% do salt['defaults.merge'](defaults['salt'], lookup) %}
```

Comment:
What is advantage of not using update() here?

Reply:
As explained in [this comment](https://github.com/saltstack-formulas/apache-formula/pull/252#issuecomment-458490196).